frontend/src/modules/controllers/ValueClicker.tsx

```diff
@@ -26,6 +30,20 @@ const useStyles = makeStyles((theme: Theme) => ({
 
 type Direction = 'column' | 'column-reverse' | 'row' | 'row-reverse' | undefined;
 
+/**
+ * @typedef Props
+ *
+ * Props interface for the value clicker
+ *
+ * @prop {number} value Paramater value displayed in UI
+ * @prop {function} onClick Callback with value on button click or Rotary encoder
+ * @prop {number} min Allowed minimum range value
+ * @prop {number} max Allowed maximum range value
+ * @prop {Direction} direction Decides styling of `ValueClicker` placed Vertically or Horizontally
+ * @prop {number} step Alarm step difference between Range (Defaults to 1)
+ * @prop {string} referenceKey Unique identifier of alarm lower/upper range
```

Comment:
Can you give an example or two of valid `referenceKey`s?

Reply:
referenceKey would be in such format`{stateKey}_LOWER` so for FiO2, it would be something like FiO2_LOWER or FiO2_HIGHER